source/extensions/filters/network/dubbo_proxy/buffer_helper.cc

```diff
@@ -0,0 +1,167 @@
+#include "extensions/filters/network/dubbo_proxy/buffer_helper.h"
```

Comment:
Seems this file is pretty similar with `buffer_helper` in thrift_proxy, is there anyway to merge those two? cc @zuercher

Reply:
Someone already started the work of extracting this in #3502, but I'm not sure if that effort is ongoing or not.

Reply:
#4344 extracts the commonality.

Reply:
Great, @zyfjeff do you mind waiting a couple days for #4344 and rebase on it?

Reply:
@lizan  Yes, i do

Reply:
#4344 is merged, can you rebase this PR based on them?

Reply:
@lizan I will take the time to rebase recently.